stable/aws-calico/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 description: A Helm chart for installing Calico on AWS
 website: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
 name: aws-calico
-version: 0.2.2
+version: 0.3.0
```

Comment:
set this to 0.3.1 since I already merged https://github.com/aws/eks-charts/pull/228. We'll cut a release of the 0.3.1 calico chart with this included as well as the fix from 228

Reply:
I merged the upstream back in my branch, so it is at 0.3.1 now.